### PR TITLE
Pakhai viktor hw7

### PR DESCRIPTION
**Pull request by ViktorPakhai**

### CODE REVIEW

File: pakhai/HW7/task_1_hw7.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+data_string = "Create 22 a dictionary 33 in which                the keys 7777777 are numbers 333 and the values are their string interpretation. "
+
+newstring = data_string[0]
+print(newstring)
+
+for char in data_string[1:]:
+    if char != newstring[-1]:
+        newstring += char   
+
+print("Original string:   ", data_string, "\nString after removing:  ", newstring)
```
Comment (katemalash) [inline pakhai/HW7/task_1_hw7.py:13]: Good way.

File: pakhai/HW7/task_2_hw7.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+str_one = "ABCDabcd7546"
+str_two = "CDcdJHhj4623"
+
+set_one = set(str_one)
+set_two = set(str_two)
+
+result_one = set_one.union(set_two)
+result_two = set_one.symmetric_difference(set_two)
+print("Task 1:  ", sorted(result_one), "\nTask 2: ", sorted(result_two))
```
Comment (katemalash) [inline pakhai/HW7/task_2_hw7.py:13]: This is the optimal way!

File: pakhai/HW7/task_3_hw7.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+"Sweden":"Stockholm",
+"United Kingdom":"London"}
+
+for key, value in country.items():
+    print(value)
+       
+val = input("\nEnter the city: ")
+for key, value in country.items():
+         if val == value:
+             print( "\nCiti in the country - " ,key)
```
Comment (katemalash) [inline pakhai/HW7/task_3_hw7.py:27]: It was required to use the list of cities for each country, not one city

File: pakhai/HW7/task_4_hw7.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+lst = string.split()
+new_dict =dict()
+
+for items in lst:
+    if items in new_dict:
+        new_dict[items] += 1
+    else:
+        new_dict[items] = 1
+print("Result: ", new_dict)
```
Comment (katemalash) [inline pakhai/HW7/task_4_hw7.py:15]: Good Decision.

**Review by katemalash — CHANGES_REQUESTED** [review]
Please improve task 3.

File: pakhai/HW7/task_3_hw7.py (hunk 3 above)
Comment (ViktorPakhai) [reply pakhai/HW7/task_3_hw7.py]: I corrected the code

File: pakhai/HW7/task_3_hw7.py [hunk 5]
```diff
@@ -1,27 +1,21 @@
...
     print(value)
-       
+
 val = input("\nEnter the city: ")
-for key, value in country.items():
-         if val == value:
-             print( "\nCiti in the country - " ,key)
+for i in country:
+    if val in country[i]:
+        print('\nCiti in the country - ', i)
```
Comment (katemalash) [inline pakhai/HW7/task_3_hw7.py:21]: Looks better now

**Review by katemalash — APPROVED** [review]
Looks good now. You can merge pull-request.

**Pull request merged**